setup.py

```diff
@@ -51,15 +51,15 @@ def main():
           download_url='https://github.com/apel/ssm/releases',
           license='Apache License, Version 2.0',
           install_requires=[
-              'cryptography==3.3.2',
-              'stomp.py<5.0.0',
-              'python-ldap<3.4.0',
+              'cryptography',
+              'stomp.py',
+              'python-ldap',
               'setuptools',
-              'pyopenssl >=19.1.0, <=21.0.0',
+              'pyopenssl',
           ],
           extras_require={
-              'AMS': ['argo-ams-library', 'certifi<2020.4.5.2', ],
-              'daemon': ['python-daemon<=2.3.0', ],
+              'AMS': ['argo-ams-library', ],
```

Comment:
Is `certifi` not needed anymore?

Reply:
It's a dependency of `argo-ams-library`  which should normally take care of it's own dependencies. We only added it in here so that we could pin the version to one that was still Python 2 compatible.